### ISSUE TITLE
Using SimulatedFoldedBatchNorm

### ISSUE THREAD

**Issue by yiminchi** [NervanaSystems/distiller#397]
Hi,

Thanks for excellent work on Distiller! I have used example code of Quantization-Aware Training, and figured out that it quantized the weight to fixed point. However, we usually fold the batchnorm layer to convolutional layer in hardware implement. If I fold the batchnorm layer parameters to convolutional layer's weights, it will change back to the floating point, which is not expected.

I found the function "SimulatedFoldedBatchNorm" that might be help. Is there have any example for this function?

Thank you!

**Comment by guyjacob** [comment]
Indeed, this module was implemented, but sadly we haven't integrated it yet into the quantization-aware training flow.

As you may know, the release of built-in quantization functionality in PyTorch is imminent. It will include simulated BN folding during training. At this stage, it's more likely that we will wait and integrate with the built-in PyTorch APIs instead of investing more in the existing code.

Apologies if this causes any inconvenience.
